iosApp/iosApp/ContentView.swift

```diff
@@ -167,6 +167,12 @@ struct ContentView: View {
                         nearbyVM.goBack()
                     }
                 }, content: coverContents)
+                .onAppear {
+                    viewportProvider.updateCameraState(locationDataManager.currentLocation)
+                }
+                .onChange(of: locationDataManager.currentLocation) { location in
+                    viewportProvider.updateCameraState(location)
+                }
```

Comment:
question: I'm a little nervous about this not not directly testing this onChange behavior since if it is accidentally removed at some point, we might not spot that right away in our day-to-day use of the app.

Would it be feasible to add a test that prevents any future regressions here?

Reply:
[this HomeMapView test](https://github.com/mbta/mobile_app/blob/296f9a4fcaea015106b354b507351bff91885520/iosApp/iosAppTests/Views/HomeMapViewTests.swift#L95) involves changing the mocked locationDataManager's current location. That is definitely a simpler test set up than ContentView, so OK if it isn't really possible to cover this with the test with the limitations of ViewInspector - I would just more strongly suggest including the comment suggested above